### ISSUE TITLE
Ductus venosus agenesis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7363]
**Preferred term label:**
Ductus venosus agenesis

**Synonyms**
Agenesis of the ductus venosus

**Definition (free text, please give PubMed ID)**
A developmental defect characterized by lack of formation of the ductus venosus (a shunt that allows oxygenated blood in the umbilical vein to bypass the liver in the fetal circulation). Agenesis of the ductus venosus can manifest in two different morphologic patterns: The umbilical vein drains exclusively into the left branch of the intrahepatic portal vein or the umbilical vein drains into the inferior vena cava or directly into the right atrium by-passing the liver completely.
PMID: 10765517
**Parent term (use hpo.jax.org/app)**
Abnormality of the fetal cardiovascular system HP:0010948

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by DominicIliescu** [comment]
Ductus venosus (DV) is supposed to play an essential role in fetal circulation because it allows the oxygenated placental blood to bypass the liver and enables preferential flow via the right atrium and foramen ovale to the left atrium. Furthermore, normal umbilical flow via DV is involved in the normal development of the intrahepatic portal venous system (IHPVS)1. 
The lack of ‘critical anastomoses’ between the portal and umbilical venous system and hepatic–systemic venous system results in agenesis of ductus venosus (ADV) and shunting of umbilical blood through an aberrant vessel. Between 1 in 2532 to 1 in 556 fetuses had various incidences of ADV2,3. 
Two main types of ADV have been described, depending on the course and connection of the umbilical vein (UV). The umbilical-portal-hepatic shunting type is associated with intrahepatic drainage to the portal sinus (PS) or a hepatic vein (HV). The UV bypasses the liver less frequently and presents an extrahepatic course, draining into the systemic venous circulation such as the inferior vena cava (IVC) or right atrium. It rarely drains into the left atrium, iliac vein, or coronary sinus5.    
Different outcomes of ADV cases have been reported, related either to their association with other fetal abnormalities and the nuchal translucency (NT) measurement, the development of the PVS, the type of umbilical shunt, or the caliber of the shunt6-9. However, the isolated nature of the anomaly is best associated with a favorable outcome3.
The failure to develop critical anastomosis between the UV and vitelline venous system represents the link between the association of ADV and total or partial agenesis of the portal venous system. ADV and PVS anomalies have a common origin, and the prognosis is variable, given the physiological importance of the two venous systems. 
References:
1.	Chaoui R, Heling KS, Karl K. Ultrasound of the fetal veins part 1: the intrahepatic venous system. Ultraschall Med. 2014 Jun; 35 (3):208-28. 
2.	Staboulidou I, Pereira S, Cruz Jde J et al. Prevalence and outcome of absence of ductus venosus at 11(+0) to 13(+6) weeks. Fetal Diagn Ther. 2011;30(1):35-40. 
3.	Iliescu DG, Cara ML, Tudorache S, et al. Agenesis of ductus venosus in sequential first and second trimester screening. Prenat Diagn. 2014 Nov; 34(11):1099-105. 
4.	Mistinova J, Valacsai F, Varga I. Congenital absence of the portal vein--Case report and a review of literature. Clin Anat. 2010 Oct;23(7):750-8. 
5.	Strizek B, Zamprakou A, Gottschalk I, et. al. Prenatal Diagnosis of Agenesis of Ductus Venosus: A Retrospective Study of Anatomic Variants, Associated Anomalies and Impact on Postnatal Outcome. Ultraschall Med. 2019 Jun;40(3):333-339. English. 
6.	Berg C, Kamil D, Geipel A, Kohl T, et. al. Absence of ductus venosus-importance of umbilical venous drainage site. Ultrasound Obstet Gynecol. 2006 Sep;28(3):275-81. 
7.	Jaeggi ET, Fouron JC, Hornberger LK, et. al. Agenesis of the ductus venosus that is associated with extrahepatic umbilical vein drainage: prenatal features and clinical outcome. Am J Obstet Gynecol. 2002 Oct;187(4):1031-7. 
8.	Shen O, Valsky DV, Messing B, et. al. Shunt diameter in agenesis of the ductus venosus with extrahepatic portosystemic shunt impacts on prognosis. Ultrasound Obstet Gynecol. 2011 Feb;37(2):184-90. 
9.	Sau A, Sharland G, Simpson J. Agenesis of the ductus venosus associated with direct umbilical venous return into the heart--case series and review of literature. Prenat Diagn. 2004 Jun;24(6):418-23.

**Comment by monicacecilia** [comment]
### From workshop on 2022-04-05
- You can have isolated agenesis - but the definition describes the shunts between umbilical vein to vena cava or subhepatic vein. These can be associated with ductus venosus agenesis, but these are two different things. 
  - The shunts are associated malformations. 
  - Suggestion to keep agenesis of ductus venosus. But do we also have terms to describe the venous malformations associated to ductus venosus agenesis?  
- Besides Ductus venosus agenesis, there is corpus callosum agenesis. 
  - In patients with both abnormalities, they have found common variants.
  - In 2 patients with DVA, we found truncating variants in a gene called Zeb2, the diagnosis was Mowat-Wilson, but in the literature you don't find associations of DVA with a syndrome. Suggestion to also add the genes we have to look at when we have this HPO term. We want to take these two cases and see if Zeb2 has to be included as associated with DVA. 
  - DVA may be a marker for Pallister–Killian Syndrome. Not necessary for setting up the term, but to see if they have associations to some gene already. See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6681314/ 
- _“Agenesis of the ductus venosus can manifest in two different morphologic patterns: The umbilical vein drains exclusively into the left branch of the intrahepatic portal vein or the umbilical vein drains into the inferior vena cava or directly into the right atrium by-passing the liver completely. In both patterns, the preferential direction of the flow towards the foramen ovale is not present. While the first pattern leads to hyperperfusion of the liver parenchyma, the latter would result in reduced perfusion and oxygenation. Our findings suggest that agenesis of ductus venosus might induce hydrops fetalis. We conclude that in every case of hydrops fetalis the venous system should be evaluated by ultrasonography prenatally and/or immediately postnatally.”_ See https://link.springer.com/article/10.1007/s40556-020-00247-6 – This article also refers to hydrops (maybe also related with Noonan and rasopathies).
  - See also https://www.degruyter.com/document/doi/10.1515/jpm-2018-0163/html 
- DVA is an important but also very unspecific sign for a number of malformation syndromes including chromosomal abnormalities.

**Comment by pnrobinson** [comment]
See also this for disease annotation -- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6681314/

**Comment by pnrobinson** [comment]
@DominicIliescu  are these good definitions for the two subtypes?

Ductus venosus agenesis with intrahepatic drainage.
Def: A form of ductus venosus agenesis in which the umbilical vein connects via the portal sinus to the portal venous system (PVS) without giving rise to the ductus venosus. In some cases an intrahepatic shunt from a portal to a hepatic vein can be detected prenatally. This variant has been reported to have a favorable prognosis.
PMID:28934814

Ductus venosus agenesis with extrahepatic drainage and liver bypass
Def: A form of ductus venosus agenesis in which umbilical vein does not connect to the portal venous system (PVS). The liver is bypassed and the umbilical vein drains into a systemic vein, e. g. the inferior vena cava, right atrium, or rarely the left atrium, coronary sinus or iliac vein. This variant has been reported to have an unfavorable prognosis due to a presumed association with congestive heart failure and hydrops.
PMID:28934814

(Definitions adapted from Strizek B, Zamprakou A, Gottschalk I, Roethlisberger M, Hellmund A, Müller A, Gembruch U, Geipel A, Berg C. Prenatal Diagnosis of Agenesis of Ductus Venosus: A Retrospective Study of Anatomic Variants, Associated Anomalies and Impact on Postnatal Outcome. Ultraschall Med. 2019 Jun;40(3):333-339. English. doi: 10.1055/s-0043-115109. Epub 2017 Sep 21. PMID: 28934814.)

**Comment by DominicIliescu** [comment]
Yes, the definitions are correct.

I would add: 
When DV was absent, the type of the shunt was investigated. According to the prenatal classification, the shunt types were reported: type I, umbilical–systemic shunt (USS), type II, ductus venosus–systemic shunt (DVSS) and type III, portal–systemic shunt, divided into type IIIa, intrahepatic portal–systemic shunt (IHPSS) and type IIIb, extrahepatic portal–systemic shunt (EHPSS).[Achiron, R.; Kivilevitch, Z. Fetal umbilical-portal-systemic venous shunt: In-utero classification and clinical significance. Ultrasound Obs. Gynecol. 2016, 47, 739–747.Wu, H.; Tao, G.; Cong, X.; Li, Q.; Zhang, J.; Ma, Z.; Zhang, Z. Prenatal sonographic characteristics and postnatal outcomes of umbilical-portal-systemic venous shunts under the new in-utero classification: A retrospective study. Medicine 2019, 98, e14125.]. 
Recently, a new category was proposed, type IV, for cases with multiple shunts, of different types.[Nagy, R.D.; Iliescu, D.G.Prenatal Diagnosis and Outcome of Umbilical–Portal–Systemic Venous Shunts: Experience of a Tertiary Center and Proposal for a New Complex Type. Diagnostics 2022, 12, 873. https://doi.org/10.3390/diagnostics12040873]


Dominic  

    On Thursday, April 7, 2022, 05:34:01 PM GMT+3, Peter Robinson ***@***.***> wrote:  
 
 


@DominicIliescu are these good definitions for the two subtypes?

Ductus venosus agenesis with intrahepatic drainage.
Def: A form of ductus venosus agenesis in which the umbilical vein connects via the portal sinus to the portal venous system (PVS) without giving rise to the ductus venosus. In some cases an intrahepatic shunt from a portal to a hepatic vein can be detected prenatally. This variant has been reported to have a favorable prognosis.
PMID:28934814

Ductus venosus agenesis with extrahepatic drainage and liver bypass
Def: A form of ductus venosus agenesis in which umbilical vein does not connect to the portal venous system (PVS). The liver is bypassed and the umbilical vein drains into a systemic vein, e. g. the inferior vena cava, right atrium, or rarely the left atrium, coronary sinus or iliac vein. This variant has been reported to have an unfavorable prognosis due to a presumed association with congestive heart failure and hydrops.
PMID:28934814

(Definitions adapted from Strizek B, Zamprakou A, Gottschalk I, Roethlisberger M, Hellmund A, Müller A, Gembruch U, Geipel A, Berg C. Prenatal Diagnosis of Agenesis of Ductus Venosus: A Retrospective Study of Anatomic Variants, Associated Anomalies and Impact on Postnatal Outcome. Ultraschall Med. 2019 Jun;40(3):333-339. English. doi: 10.1055/s-0043-115109. Epub 2017 Sep 21. PMID: 28934814.)

**Comment by pnrobinson** [comment]
@DominicIliescu the HPO likes to separate out phenotypic features, i.e., to not "bundle" combinations of features. I wonder if it would be better to create separate terms for the four shunt types? Or are they always seen with DV agenesis, i.e., they cannot occur in other situations? I am sorry but I am not 100% sure how the four types related to the two other subtypes?

**Comment by DominicIliescu** [comment]
The term umbilical shunt refers to the absence/agenesis of ductus venosus accompanied by umbilical shunting via postal or systemic circulation. 
The intrahepatic or extrahepatic classification is simple, while the type 1 to 3 classification is more complex, but also more accurate. I would recommend that both classifications should be part of the text, as presented in the previous contribution, because the studies may refer to any of the two classifications.

Dominic Iliescu, MD, PhD, MPH
Associate Professor, University of Medicine and Pharmacy CraiovaPetru Rares nr.2, Craiova, Dolj, Romania, 200349
Chief Physician of the 2nd Clinic of Obstetrics and Gynecology, Emergency County University Hospital CraiovaTabaci nr.1, Craiova, Dolj, Romania, 200642
Phone: +40723 888 773 

    On Sunday, April 10, 2022, 02:05:42 PM GMT+3, Peter Robinson ***@***.***> wrote:  
 
 


@DominicIliescu the HPO likes to separate out phenotypic features, i.e., to not "bundle" combinations of features. I wonder if it would be better to create separate terms for the four shunt types? Or are they always seen with DV agenesis, i.e., they cannot occur in other situations? I am sorry but I am not 100% sure how the four types related to the two other subtypes?

**Comment by pnrobinson** [comment]
For now let's move forward like this, we can revise at a future workshop as needed.

Adding new term: 
- ID: HP:6000369
- label: Ductus venosus agenesis with intrahepatic drainage
- parents: [Ductus venosus agenesis (HP:0034196)](https://hpo.jax.org/app/browse/term/HP:0034196)
- pmid: PMID:28934814
- def: A form of ductus venosus agenesis in which the umbilical vein connects via the portal sinus to the portal venous system (PVS) without giving rise to the ductus venosus. In some cases an intrahepatic shunt from a portal to a hepatic vein can be detected prenatally. This variant has been reported to have a favorable prognosis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7363
- synonyms: 


and

Adding new term: 
- ID: HP:6000370
- label: Ductus venosus agenesis with extrahepatic drainage and liver bypass
- parents: [Ductus venosus agenesis (HP:0034196)](https://hpo.jax.org/app/browse/term/HP:0034196)
- pmid: PMID:28934814
- def: A form of ductus venosus agenesis in which umbilical vein does not connect to the portal venous system. The liver is bypassed and the umbilical vein drains into a systemic vein such as the inferior vena cava. This variant has been reported to have an unfavorable prognosis due to a presumed association with congestive heart failure and hydrops.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7363
- synonyms:
